components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/pom.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ WSO2 Inc. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.local.auth.fido</groupId>
+        <artifactId>identity-application-auth-fido</artifactId>
+        <relativePath>../../pom.xml</relativePath>
+        <version>5.1.22-SNAPSHOT</version>
+    </parent>
+
+    <packaging>war</packaging>
+    <artifactId>org.wso2.carbon.identity.application.authenticator.fido2.endpoint</artifactId>
+    <name>WSO2 Carbon - FIDO2 Identity Application Endpoint</name>
+
+    <build>
+        <plugins>
+            <!-- Plugin for generating the swagger server stubs-->
+            <!--<plugin>
```

Comment:
Why we are having commented line? shall we remove this?

Reply:
since this plugin is required only if it is needed to regenerate the swagger api definition, this plugin has been commented.